### ISSUE TITLE
Test set for comparison 

### ISSUE THREAD

**Issue by givkashi** [fenglinglwb/MAT#14]
Hello, you reported results on CelebA-HQ at 256 × 256 size in Table F.3. what is your test set and how we can access it for comparison?
How did you use Places (512 × 512) to train and test the model?
@fenglinglwb

**Comment by fenglinglwb** [comment]
Thanks for your attention to our work. In two days, I will upload the model and testing masks for CelebA-HQ at 256 × 256 sizes.
As for Places, we trained the model using the train set in Places365-Standard (1.8M images, URL: http://data.csail.mit.edu/places/places365/train_large_places365standard.tar) and used the validation set (36500 images, URL:http://data.csail.mit.edu/places/places365/val_large.tar) for evaluation.

**Comment by givkashi** [comment]
Thanks. Please say how you prepared the test set for Places (512 × 512) in the small mask and large mask that you report results in table 2.

**Comment by fenglinglwb** [comment]
The small and large masks for Places validation images are already given in this repo.
![image](https://user-images.githubusercontent.com/12084739/169565685-86a1e378-8f19-4c35-970f-7f4b815c8fa1.png)

**Comment by fenglinglwb** [comment]
Sorry for the delay. I will upload the testing masks and model this weekend.
